### PR TITLE
Support Ubuntu 18.04

### PR DESCRIPTION
**Pull request by manics**
In theory omero-web Python 3 should be quite easy to install on Ubuntu with Ansible now that it mostly involves pip installing things. This is the only remaining role dependency that's doesn't yet support Ubuntu. The other dependencies are either converted, aren't needed for Python 3 omero-web, or are no-ops on Ubuntu (selinux).

Tag: `2.0.0` (bumped the minimum version of Ansible to 2.6)

Updated: omero-web on ubuntu:18.04: https://github.com/manics/ansible-role-omero-web/commit/748a667c009c8518ea6265f487a21a3560c392af

### CODE REVIEW

File: molecule/ubuntu1804/molecule.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+platforms:
+  - name: nginx-default
+    image: leandelivery/docker-systemd:ubuntu-18.04
```
Comment (sbesson) [inline molecule/ubuntu1804/molecule.yml]: I haven't investigated but I suspect this means there is no official Ubuntu Docker image with systemd support?

File: meta/main.yml [hunk 2]
```diff
@@ -4,11 +4,14 @@ galaxy_info:
   description: Install upstream Nginx
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.6
```
Comment (sbesson) [inline meta/main.yml]: Is the bump here related to a particular feature or the fact 2.6 is our current production Ansible version?

File: molecule/ubuntu1804/molecule.yml (hunk 1 above)
Comment (manics) [reply molecule/ubuntu1804/molecule.yml]: No official images, this one was mentioned on https://github.com/ansible/molecule/issues/1768

File: meta/main.yml (hunk 2 above)
Comment (manics) [reply meta/main.yml]: We haven't tested 2.1 for ages and this contains some big changes.

Comment (manics) [reply meta/main.yml]: `import_tasks` requires 2.4 https://docs.ansible.com/ansible/2.4/import_tasks_module.html

**Pull request merged**